src/scss/layout/_header.scss

```diff
@@ -1,3 +1,68 @@
-.header {
-  height: 90px;
+@mixin  fontStyle ($color, $fontSize) {
+    color: $color;
+    font-size: $fontSize;
+    font-family: Helvetica;
+    font-style: normal;
+    font-weight: bold;
 }
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+header {
```

Comment:
The content of the header should expand only until 1440px. You can use a max-width attribute or use the class 'component' which will implement the layout later.

Reply:
Please use a class instead of the tag as selector: `.header`. Also add the class to the header element.

Reply:
Thanks you. Done.